deployments/scroll.diamond.json

```diff
@@ -92,14 +92,23 @@
       "0x6F2baA7cd5F156CA1B132F7FF11E0fa2aD775F61": {
         "Name": "EmergencyPauseFacet",
         "Version": "1.0.0"
+      },
+      "0xF5c923a087fb3c554579e2DD10AB6E37E0f6F849": {
+        "Name": "",
+        "Version": ""
+      },
+      "0x6e00e0a7685Ca22c288d56D9E7924746B5043Ee7": {
+        "Name": "AcrossFacetV3",
+        "Version": "1.0.0"
       }
     },
     "Periphery": {
       "ERC20Proxy": "0xA950Ac46b0b844c0564d18A54A9685e614B9086C",
       "Executor": "0x7078d1DE45C7D3e87f71D5DA663db2a8Ee1dfEbe",
       "FeeCollector": "0xF048e5816B0C7951AC179f656C5B86e5a79Bd7b5",
-      "LiFiDEXAggregator": "0x6140b987d6B51Fd75b66C3B07733Beb5167c42fc",
+      "LiFiDEXAggregator": "0x78bF01555bCF05e6B1d4dad017dBD0A105652DC9",
       "LiFuelFeeCollector": "0xc02FFcdD914DbA646704439c6090BAbaD521d04C",
+      "Permit2Proxy": "",
```

Comment:
the Permit2Proxy is not deployed on this network, otherwise you would find its address in the file deployments/scroll.json

So for the future please only point such issues out if you find a contract's address in this deployments/<network>.json but not in the file deployments/<network>.diamond.json